### PR TITLE
Added tests and database dump. fixed bugs

### PR DESCRIPTION
**Pull request by egrossman**
@dlrobertson  can you tell me how to run the nosetests? I still need to check if these tests work.

### CODE REVIEW

**Comment by bmenn** [comment]
@egrossman 

You can run `nosetests` by running

```
python setup.py nosetests
```

You may need to make sure all the necessary packages are installed before running the above

```
pip install -e .[tests]
```

Another trick is to take a look at the CI build instructions (in this case, `.travis.yml`).

**Comment by egrossman** [comment]
@bmenn got the nosetests to work. Had to use a different assert method for the dataframes

File: htetl/extract/phones.py [hunk 1]
```diff
@@ -50,11 +51,29 @@ def _norm(text):
 
     return '-'.join([norm_area_code, norm_first_three, norm_last_four])
 
+
+def ExtractPhones(df):
```
Comment (bmenn) [inline htetl/extract/phones.py]: PEP-8 nit. Should rename to `extract_phone`

File: htetl/extract/phones.py [hunk 2]
```diff
@@ -50,11 +51,29 @@ def _norm(text):
...
+        if match:
+            for phone in match:
+                pnumber=normalize(phone)
+
+                phones.append([row["id"],pnumber])
+
+    phonedf = pd.DataFrame(phones,columns=["pageid","phone"]).drop_duplicates()
+    return phonedf
+
 class ParsePhones(luigi.Task):
```
Comment (bmenn) [inline htetl/extract/phones.py]: Maybe move this into it's own module under `tasks`?

**Review by bmenn — CHANGES_REQUESTED** [review]
2 small nitpicks. Looks good otherwise.

File: htetl/extract/phones.py (hunk 1 above)
Comment (dlrobertson) [reply htetl/extract/phones.py]: I changed this to `extract_phone` PEP-8 is snake case right?

Comment (bmenn) [reply htetl/extract/phones.py]: yes, you are correct. I apparently should go sleep (writing Python is my day job).

File: htetl/extract/phones.py (hunk 2 above)
Comment (egrossman) [reply htetl/extract/phones.py]: The entire method or just the conversion to the dataframe? Extract the phone numbers in one task and normalize it in another?

Comment (bmenn) [reply htetl/extract/phones.py]: I'd move the class `ParsePhones` under `tasks`, similar to how I organized the external site extraction.

**Review by dlrobertson — APPROVED** [review]

**Review by bmenn — APPROVED** [review]

**Pull request merged**